util/pyclient/replica_specific_info.py

```diff
@@ -0,0 +1,82 @@
+# Concord
+#
+# Copyright (c) 2019 VMware, Inc. All Rights Reserved.
+#
+# This product is licensed to you under the Apache 2.0 license (the "License").
+# You may not use this product except in compliance with the Apache 2.0 License.
+#
+# This product may include a number of subcomponents with separate copyright
+# notices and license terms. Your use of these subcomponents is subject to the
+# terms and conditions of the subcomponent's license, as noted in the LICENSE
+# file.
+
+# This code requires python 3.5 or later
+
+from collections import namedtuple
+import bft_msgs
+
+CommonReplyHeader = namedtuple('CommonReplyHeader', ['span_context_size', 'primary_id',
+                                                     'req_seq_num', 'length'])
+
+MatchedReplyKey = namedtuple('MatchedReplyKey', ['req_seq_num', 'header', 'data'])
+
+
+class MsgWithSpecificReplicaInfo:
```

Comment:
Please stick to the conventional name: `ReplicaSpecificInfo` rather than `SpecificReplicaInfo`

Reply:
done